client/api/src/cht.rs

```diff
@@ -94,9 +95,8 @@ where
 	I: IntoIterator<Item = ClientResult<Option<Header::Hash>>>,
 {
 	use sp_trie::TrieConfiguration;
-	Ok(sp_trie::trie_types::Layout::<Hasher>::trie_root(build_pairs::<Header, I>(
-		cht_size, cht_num, hashes,
-	)?))
+	Ok(sp_trie::Layout::<Hasher>::default()
```

Comment:
Not that layout default is not the trie with inner hashing, so should be use in many case: cht, extrinsic root, and others.